packages/atti-components/src/components/Toggle/Readme.mdx

```diff
@@ -0,0 +1,58 @@
+---
+path: '/toggle'
+name: 'Toggle'
+title: 'Toggle'
+---
+
+Toggle offers the user the possibility to choose or not item.
+
+## Example
+
+```js
+<div
+  style={{
+    paddingBottom: '30px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    margin: '0 48px 0 -10px',
```

Comment:
I was wondering why you had to use this weird negative margin.
And finally it's because the first DOM element of the Toggle is in `position: absolute;` which is not normal and may create a lot of issues when using it.
You'd better change it for a `position: relative;` (so that its children in absolute position will refer to it) + change it's display to something like `display: inline-block;` because right now it's a span (display is inline) and so width / height properties won't get any effect on it ;).

Reply:
Thanks for the advice. I've rectified that :).

Reply:
👌 Perfect